src/it/resources/org/checkstyle/suppressionxpathfilter/booleanexpressioncomplexity/SuppressionXpathRegressionBooleanExpressionComplexityOne.java

```diff
@@ -0,0 +1,15 @@
+package org.checkstyle.suppressionxpathfilter.booleanexpressioncomplexity;
+
+public class SuppressionXpathRegressionBooleanExpressionComplexityOne {
+                public boolean method() {
+                    boolean a = true;
```

Comment:
please make indentaiton good, no reason to make it weird in this test.
lets have it good to read for humans

Reply:
Sure sir !